package.json

```diff
@@ -42,7 +45,8 @@
     "ssb-msgs": "^5.2.0",
     "ssb-ref": "^2.13.9",
     "ssb-thread-schema": "^1.1.1",
-    "yargs": "^15.0.0"
+    "yargs": "^15.0.0",
+    "sharp": "^0.23.0"
```

Comment:
I'm conflicted about this change from `npm init`. The `optionalDependencies` property overrides `dependencies`, so this won't cause any change to our behavior, but the docs say:

> Entries in **`optionalDependencies`** will override entries of the same name in **`dependencies`**, so it’s usually best to only put in one place.

Why does `npm init` do this?!

Reply:
tbh I'm slightly annoyed but also totally fine with the behavior -- it's weird, but I'm fine with npm defaults as long as that's what we're using :man_shrugging: